Core/DefInjected/QuestScriptDef/Script_BanditCamp.xml

```diff
@@ -25,13 +25,12 @@
     <li>bandit->Brigand</li>
   -->
   <OpportunitySite_BanditCamp.questNameRules.rulesStrings>
-    <li>questName->[bandit] [camp]</li>
-    <li>questName->[bandit] [camp]</li>
-    <li>questName->[asker_nameDef] and the [camp]</li>
-    <li>camp->Kamp</li>
-    <li>camp->Nest</li>
-    <li>camp->Schuilplaats</li>
-    <li>camp->Kampement</li>
+    <li>questName->[camp] van de [bandit]</li>
+    <li>questName->[asker_nameDef] en [camp]</li>
+    <li>camp->het Kamp</li>
```

Comment:
Ik vind het een beejte vreemd dat deze woorden met hoofdletter zijn geschreven? Ik zie dat het al zo was in de vorige vertaling, maar vraag me af of het er goed uitziet in de game?